examples/cookbooks/Industry_Templates/healthcare_template.py

```
@@ -0,0 +1,478 @@
+            "status": "completed",
+            "data": vital_signs
+        })
+        
+        # Safety check: Validate vital signs are within possible ranges
+        if vital_signs.get("heart_rate", 0) < 20 or vital_signs.get("heart_rate", 0) > 300:
+            workflow_result["safety_checks"].append({
+                "type": "vital_signs_validation",
+                "issue": "impossible_heart_rate",
+                "action": "manual_verification_required"
+            })
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)